### ISSUE TITLE
Browser check

### ISSUE THREAD

**Issue by dgorissen** [taarifa/TaarifaWaterpoints#55]
When loading app (in particular the dashboard) give a warning if using an outdated browser

**Comment by mtuchi** [comment]
Does this mean checking also in different device as well like smartphone and tablets?

**Comment by kynan** [comment]
That's right, we're particularly interested in the mobile browsing experience.
